### ISSUE TITLE
Tab completion issue

### ISSUE THREAD

**Issue by SladeHazard** [PaperMC/Velocity#533]
Where can I disable velocity sending commands to players when they tab complete?
https://i.ibb.co/XLyFgXx/image.png

**Comment by hugmanrique** [comment]
You can set `announce-proxy-commands` to false in the velocity config file, but it's better practice to disallow players from using the commands by negating the corresponding permission nodes.

Note partial alias suggestions don't respect player permissions on 1.1.x. 3.0 includes a fix for this.

**Comment by SladeHazard** [comment]
I already have that set to false it's only for 1.13+ clients. players already don't have perms for those cmds

**Comment by hugmanrique** [comment]
> Note partial alias suggestions don't respect player permissions on 1.1.x. 3.0 includes a fix for this.

Try updating to 3.0 (download the latest build from https://ci.velocitypowered.com/job/velocity-3.0.0/) and see if that fixes the issue.

**Comment by SladeHazard** [comment]
> > Note partial alias suggestions don't respect player permissions on 1.1.x. 3.0 includes a fix for this.
> 
> Try updating to 3.0 (download the latest build from https://ci.velocitypowered.com/job/velocity-3.0.0/) and see if that fixes the issue.

Did not fix it.

**Comment by astei** [comment]
Not sure why the permissions aren't being applied correctly, but due to recent command API implementation changes, I was able to make `announce-proxy-commands` work with 1.12.2 and below with 3b6b73f. This should be sufficient until the bigger issue is fixed.

**Comment by hugmanrique** [comment]
I've tried build [32](https://ci.velocitypowered.com/job/velocity-3.0.0/32/) in the CI with `announce-proxy-commands` set to `true` and everything works as expected in 1.12.2 and in 1.16.5. This is what I did:

1. Run `lp user <your_username> set velocity.command.server true` in the proxy console (this is not strictly necessary since it's the default, but it's here for completeness)
2. Log in with `<your_username>` and observe that `/ser` suggests `/server` as expected.
3. Run `lp user <your_username> set velocity.command.server false` in the proxy console.
4. If the client is on >=1.13, relog.
5. Observe that `/`, `/ser` and others no longer suggest `/server`.
